packages/manager/apps/sms/src/index.js

```diff
@@ -1,5 +1,9 @@
-import 'script-loader!jquery'; // eslint-disable-line
-import 'script-loader!lodash'; // eslint-disable-line
+/* eslint-disable import/no-webpack-loader-syntax */
```

Comment:
I take the opportunity to explicitly refer to the rule's name which need to be turned off.